source/dotnet/Samples/WPFVisualizer/AdaptiveCards.Sample.WPFVisualizer.csproj

```diff
@@ -148,6 +162,21 @@
       <SubType>Designer</SubType>
     </None>
   </ItemGroup>
+  <ItemGroup>
+    <Content Include="..\..\..\..\samples\HostConfig\**">
+      <Link>HostConfigs\%(Filename)%(Extension)</Link>
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

Comment:
>Always [](start = 29, length = 6)

Here and below we should be using `PreserveNewest` (helps speed up the build)

Reply:
Sounds good 👍 updating now along with figuring out why the .NET CI build didn't succeed!